sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -282,4 +283,36 @@ export class EducationProgramOfferingController {
       description: offering.name,
     }));
   }
+
+  /**
+   * Gets program offering date
+   * @param locationId location id.
+   * @param programId program id.
+   * @param offeringId offering id
+   * @returns offering date for the given offering
+   */
+  @AllowAuthorizedParty(AuthorizedParties.student)
+  @Get(
+    "location/:locationId/education-program/:programId/offering/:offeringId/date",
```

Comment:
I would avoid creating a specific method down this level. Right now we need the date but in the future, we can need any other property, and another and another 😄 
We have a method to get the offering by Id for Institutions, and we can have a similar one for Students.
_location/:locationId/education-program/:programId/offering/:offeringId_

Reply:
okay, maybe we can use the same function, i can update "location/:locationId/education-program/:programId/offering/:offeringId/date" to "location/:locationId/education-program/:programId/offering/:offeringId/details". does that make sense @andrewsignori-aot

Reply:
The current state of the PR looks good.